api/admin/controller/discovery_service_library_registrations.py

```diff
@@ -61,7 +61,7 @@ def process_get(self):
                     id=registry.integration.id,
                     libraries=libraries,
                 )
-            )
+            
```

Comment:
Did you intend to make this change? Seems like it's going to introduce a syntax error.

Reply:
No, that was an accident; fixed it!